keps/sig-node/3619-supplemental-groups-policy/README.md

```diff
@@ -948,6 +1050,18 @@ This through this both in small and large cases, again with respect to the
 
 No.
 
+###### Can enabling / using this feature result in resource exhaustion of some node resources (PIDs, sockets, inodes, etc.)?
+
+<!--
+Focus not just on happy cases, but primarily on more pathological cases
+(e.g. probes taking a minute instead of milliseconds, failed pods consuming resources, etc.).
+If any of the resources can be exhausted, how this is mitigated with the existing limits
+(e.g. pods per node) or new limits added by this KEP?
+
+Are there any tests that were run/should be run to understand performance characteristics better
+and validate the declared limits?
+-->
```

Comment:
Answer here should be "No" I think?

Reply:
+1

should be No

Reply:
```suggestion
-->

No
```

Reply:
I think it's OK to merge without this answer to hit the KEP deadline